source/includes/_resource.customers.md

```diff
@@ -503,7 +560,7 @@ $ curl -X GET "https://api.vhx.tv/customers/1/watching" \
         <span class="is-block text--transparent text-3">string</span>
         <span class="text--yellow text-3">REQUIRED</span>
       </td>
-      <td>The <code>href</code> of the customer being retrieved. This needs to be provided in the request url and the <code>VHX-Customer</code> header.</td>
+      <td>The <code>href</code> of the customer being retrieved must be passed in via the <code>VHX-Customer</code> header.</td>
```

Comment:
Should we update the `Example Response` specifically to include mention of the `browse_url` with the authenticated token?

Reply:
Which requests do we get that back for? Any customer resource?

Reply:
Nvm, I found it. It looks like you can get it from the basic retrieve customer endpoint. Its found under embedded > products > links > browse_page > href. It think it would be hard to display such a nested item in the example. We could just provide that in one of the endpoint or the resource description. This might be worth confirming with the platform team that we want this publicly available instead of maybe sent in a more private way to enterprise clients.

Reply:
I think it makes sense to include it in the retrieve customer section. Maybe just underneath the response we include `*the _embedded object contains....`

Reply:
Like under the example response in the code snippet section? Or just like inline with the response under the param options?

Reply:
I was picturing under the example response, but fine with whatever you think looks best